src/cf.data.pre

```diff
@@ -1071,8 +1071,9 @@ DEFAULT: all src all
 DEFAULT: manager url_regex -i ^cache_object:// +i ^[^:]+://[^/]+/squid-internal-mgr/
 DEFAULT: localhost src 127.0.0.1/32 ::1
 DEFAULT: to_localhost dst 127.0.0.0/8 0.0.0.0/32 ::1/128 ::/128
+DEFAULT: to_APIPA dst 169.254.0.0/16
```

Comment:
We should be matching link-local IPv6 addresses as well, right? I think they are covered by RFC 2462 which mentions `fe80::0` prefix for those, but AWS [uses](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-metadata.html) a ULA `fd00:ec2::254` address instead. Should we add something like the following?

```suggestion
DEFAULT: to_APIPA dst 169.254.0.0/16 fe80::/10
```

Reply:
Suggest adding `fd00:ec2::254/128` for EC2.

Reply:
Also, `DEFAULT: to_linklocal` would work better IMO. I've learned that "APIPA" is outmoded terminology recently.